www/i18n/en.json

```diff
@@ -307,27 +307,28 @@
           "android-10": "Please select 'Allow all the time'",
           "android-11": "On the app settings page, choose the 'Location' permission and set it to 'Allow all the time'",
           "android-gte-12": "On the app settings page, choose the 'Location' permission and set it to 'Allow all the time' and 'Precise'",
-          "ios-lt-13": "Please select 'Always allow'",
-          "ios-gte-13": "On the app settings page, please select 'Always' and 'Precise' and return here to continue"
+          "ios-lt-13": "Select 'Always Allow' to enable location tracking",
+          "ios-13-13.3": "Select 'Always' location permissions on the next screen to enable location tracking, then return here to continue",
+          "ios-gte-13.4": "Select 'Precise' and 'Allow While Using App' on the next screen, then 'Change to Always Allow' to enable location tracking"
         }
       },
       "overall-fitness-name-android": "Physical activity",
       "overall-fitness-name-ios": "Motion and Fitness",
-      "overall-fitness-description": "The fitness sensors distinguish between walking, bicycling and motorized modes.  We use this data in order to separate the parts of multi-modal travel such as transit. We also use it to as a cross-check potentially spurious trips - if the location sensor jumps across town but the fitness sensor is stationary, we can guess that the trip was invalid.",
+      "overall-fitness-description": "The fitness sensors distinguish between walking, bicycling and motorized modes. We use this data in order to separate the parts of multi-modal travel such as transit. We also use it to as a cross-check potentially spurious trips - if the location sensor jumps across town but the fitness sensor is stationary, we can guess that the trip was invalid.",
       "fitnessperms": {
         "name": "Fitness Permission",
         "description": {
           "android": "Please allow.",
-          "ios": "Please allow."
+          "ios": "Allow motion and fitness tracking to enable mode detection"
         }
       },
       "overall-notification-name": "Notifications",
-      "overall-notification-description": "We need to use notifications to inform you if the settings are incorrect. We also use hourly invisible push notifications to wake up the app and allow it to upload data and check app status. We also use notifications to remind you to label your trips.",
+      "overall-notification-description": "We use notifications to inform you if the settings are incorrect. We also use hourly invisible push notifications to wake up the app and allow it to upload data and check app status. We also use notifications to remind you to label your trips.",
```

Comment:
we may be able to remove this "We also use hourly invisible push notifications to wake up the app and allow it to upload data and check app status." now that we know that we don't need the notification permission for it.